nectar/downloaders/threaded.py

```diff
@@ -319,7 +319,7 @@ def _fetch(self, request):
             self.fire_download_progress(report)
 
         except SkipLocation:
-            _logger.debug("Skipping {url} because {netloc} could not be reached.".format(
+            _logger.info("Skipping {url} because {netloc} could not be reached.".format(
```

Comment:
If a netloc is in a blacklist, should we just skip logging it or should I have the code skip trying it altogether?